product_docs/docs/pem/8/changing_default_port.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Changing the default port"
+---
+
+The PEM Server is configured after the installation either automatically (in Windows) or manually by running the configuration script (in Linux). While configuring the PEM Server, by default the 8443 port is assigned for the web services. You can change the port as per your requirement after configuration by changing a few parameters in the apache configuration files. The apache configuration files names and locations are platform specific.
```

Comment:
use lowercase for server when referring to the PEM server throughout these changes.

Reply:
I did all changes except the last sentence "They differ for RHEL and CentOS versus Debian 10 and Ubuntu 18 or 20" as more platforms will be added as and when tested